scenario/README.md

```diff
@@ -0,0 +1,108 @@
+# Scenarios
+
+**Inductiva API** provides a framework for users to create personalized scenarios. Scenarios describe a physical model of a process in nature, that can be simulated. With a scenario, the complexity of configuring the simulators is abstracted and allows users to configure only a few parameters to make an exploration of the physical model.
+
+As a **user**, you will have the ability to create your own scenarios. A few fundamental blocks are required to create a scenario:
+- Choose the simulators you want to use;
+- Template the input files for the simulators;
+- Construct the Python code based on Inductiva API.
+
+In this repository, we present two examples of scenarios that simulate the motion of fluid in a cubic tank: the `DamBreak` and the `FluidBlock` scenarios.
+
+The scenarios are meant to **inspire** the creation of personalized scenarios.
+Together, with the scenarios we also show how the outputs can be handled by post-processing methods and visualize the results.
+
+To learn more on how to create your scenarios, check the [documentation](https://github.com/inductiva/inductiva/wiki/Build-your-own-scenario).
+
+### Dam break scenario
+
+This scenario simulate the motion of a fluid block under the effect of gravity.
+The simulators available are [SPlisHSPlasH](https://github.com/inductiva/inductiva/blob/main/inductiva/simulators/splishsplash.py) and [DualSPHysics](https://github.com/inductiva/inductiva/blob/main/inductiva/simulators/dualsphysics.py), which use the [Smoothed Particle Hydrodynamics](https://en.wikipedia.org/wiki/Smoothed-particle_hydrodynamics) formulation to model the fluid.
+
+One can initialize the scenario with different fluids (Water, Honey, Olive oil) and with fluid blocks of different dimensions and initial positions. Then, users can further configure the simulation time and resolution.
+
+##### Example
+
+We have prepared a Python script to directly run this scenario. Get inside the scenario folder and run the following command:
+
+```bash
+python3 dam_break_run.py
+```
+
+Or, prepare your own and start changing the parameters. The script should be inside the scenario folder. Copy the code snippet and get started:
+
+```python
+import dam_break
+import post_processing
+from utils import fluid_types
+
+# Initialize the DamBreak scenario
+dam_break = dam_break.DamBreak(fluid=fluid_types.WATER)
+
+# Launch the simulation
+task = dam_break.simulate(
+    simulation_time=4,
+    resolution="medium")
+
+# Wait for the simulation to finish and download the results
+task.wait()
+output_dir = task.download_outputs()
+
+# Post-process the results and render a video of the simulation
+simulation_output = post_processing.SPHSimulationOutput(output_dir)
+simulation_output.render(fps=60)
+```
+
+<p align="center">
```

Comment:
Here, I refrained to use higher resolution so that it is easy to plug-and-play without waiting to much time for the simulation (short attention span).

These are the ones we are providing via the Output class, so I will keep them here. In the main ReadME we will have a catchy simulation rendering with the low-level.